operator/pkg/manifest/shared.go

```diff
@@ -203,6 +203,9 @@ func GenIOPFromProfile(profileOrPath, fileOverlayYAML string, setFlags []string,
 	}
 	// InstallPackagePath may have been a URL, change to extracted to local file path.
 	finalIOP.Spec.InstallPackagePath = installPackagePath
+	if ns := GetValueForSetFlag(setFlags, "values.global.istioNamespace"); ns != "" {
+		finalIOP.Namespace = ns
```

Comment:
This seems like a good idea but not sure this the right way to fix this because there is some discussion going on in  https://github.com/istio/istio/issues/30897 that also points out if the CNI is used.

Reply:
cc: @morvencao

Reply:
This change will not fix the CNI issue however it at least fix current IOP state problem.